test/recipe/admin/components/Baked/Component.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <component
+    :is="is"
+    v-if="loaded"
+    :schema="descriptor.$schema"
+    :data="data"
```

Comment:
rename these as `$schema` and `$data` for all components to match them to their prop names under descriptor

Reply:
since `$data` was reserved by vue, we went back to `schema` and `data` with one exception `$path`